src/dowel/csv_output.py

```diff
@@ -25,34 +30,52 @@ def types_accepted(self):
         """Accept TabularInput objects only."""
         return (TabularInput, )
 
+
     def record(self, data, prefix=''):
         """Log tabular data to CSV."""
         if isinstance(data, TabularInput):
             to_csv = data.as_primitive_dict
-
+            print('-----')
+            print(to_csv)
             if not to_csv.keys() and not self._writer:
                 return
 
-            if not self._writer:
-                self._fieldnames = set(to_csv.keys())
-                self._writer = csv.DictWriter(
-                    self._log_file,
-                    fieldnames=self._fieldnames,
-                    extrasaction='ignore')
-                self._writer.writeheader()
-
-            if to_csv.keys() != self._fieldnames:
-                self._warn('Inconsistent TabularInput keys detected. '
-                           'CsvOutput keys: {}. '
-                           'TabularInput keys: {}. '
-                           'Did you change key sets after your first '
-                           'logger.log(TabularInput)?'.format(
-                               set(self._fieldnames), set(to_csv.keys())))
-
+            # if not self._writer:
```

Comment:
Generally, we just delete unused code, instead of commenting it out. It will still be in the git history if we need it :)